app/src/main/java/com/work/restaurant/GoogleMapFragment.kt

```diff
@@ -0,0 +1,120 @@
+package com.work.restaurant
+
+import android.content.Context
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.MapView
+import com.google.android.gms.maps.OnMapReadyCallback
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+
+
+class GoogleMapFragment : OnMapReadyCallback, Fragment() {
+
+
+    private lateinit var mapView: MapView
+
+    override fun onMapReady(p0: GoogleMap) {
+
+        val SEOUL = LatLng(37.56, 126.97)
```

Comment:
```suggestion
        val seoulPosition = LatLng(37.56, 126.97)
```
상수가 아니니 변수명으로 작성하는게 좋을 것 같아요.

Reply:
5bf0004 수정하였습니다.